### PR TITLE
Add Warn Broken Links build option

### PR DESCRIPTION
**Pull request by TrushynAliaksandr**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -12,21 +12,23 @@ steps:
...
         hint: "Should all books be rebuilt, regardless of what has changed? Build once with this set to true after every release."
-      - select: "Allow Broken Links?"
-        key: "ALLOW_BROKEN_LINKS"
+      - select: "How should broken links be handled?"
+        key: "BROKEN_LINKS"
         default: false
         required: false
         options:
-          - label: "no"
+          - label: "skip"
```
Comment (nkammah) [inline .buildkite/pipeline.yml:20]:
```suggestion
          - label: "Continue without warning"
```

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -12,21 +12,23 @@ steps:
...
+            value: "skip"
+          - label: "warn"
```
Comment (nkammah) [inline .buildkite/pipeline.yml:22]:
```suggestion
          - label: "Continue, but log a warning"
```

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -12,21 +12,23 @@ steps:
...
+            value: "warn"
+          - label: "fail the build"
```
Comment (nkammah) [inline .buildkite/pipeline.yml:24]:
```suggestion
          - label: "Fail the build"
```

File: .buildkite/scripts/build-docs.sh [hunk 4]
```diff
@@ -13,6 +13,7 @@ export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
 build_args=""
 rebuild_opt=""
 skiplinkcheck_opt=""
+warnlinkcheck_opt=""
```
Comment (nkammah) [inline .buildkite/scripts/build-docs.sh:16]: let's streamline a bit and use a single variable for the broken_links_opt that will either have the `--skiplinkcheck` or `--warnlinkcheck` value

Comment (TrushynAliaksandr) [reply .buildkite/scripts/build-docs.sh]: fixed

**Pull request merged**